apps/mobile/src/components/wallet-manager/account-list.tsx

```diff
@@ -0,0 +1,32 @@
+import { ReactNode } from 'react';
+import { Pressable, Text, View } from 'react-native';
+
+import { Account } from '@/state/accounts/accounts';
+
+interface AccountsLayoutProps {
+  fingerprint: string;
+  accounts: Account[];
+  onRemoveAccount: (accountIndex: number) => void;
+  renderKeychains(fingerprint: string, accountIndex: number): ReactNode;
+}
+export function AccountsLayout({
+  fingerprint,
+  accounts,
+  onRemoveAccount,
+  renderKeychains,
+}: AccountsLayoutProps) {
+  return accounts.map(account => (
```

Comment:
How about de-structuring `account` to keep things DRY? 

```
return accounts.map({accountIndex, id} => (
```

Reply:
Yep, good point @alter-eggo, map moved